tests/infrastructure/node-labeller.go

```diff
@@ -203,15 +203,22 @@ var _ = DescribeInfra("Node-labeller", func() {
 		})
 
 		It("[test_id:6247] should set default obsolete cpu models filter when obsolete-cpus-models is not set in kubevirt config", func() {
+			kvConfig := v1.KubeVirtConfiguration{ObsoleteCPUModels: nil}
+			tests.UpdateKubeVirtConfigValueAndWait(kvConfig)
```

Comment:
This configuration change could affect other tests.
Shouldn't it be reverted at the end of the test?

Reply:
AFAIK it happens automatically here:

https://github.com/kubevirt/kubevirt/blob/a72353209753b177a2d71abc7204c6254befc89a/tests/tests_suite_test.go#L109

Reply:
I would prefer to retrieve the current KubeVirt config and update only the `ObsoleteCPUModels`
```
kvConfig = util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
```

`DescribeInfra` is `Serial` so, yes, the kv config is restored in global cleanup

Reply:
> I would prefer to retrieve the current KubeVirt config and update only the ObsoleteCPUModels

why?

Reply:
> I would prefer to retrieve the current KubeVirt config and update only the ObsoleteCPUModels
Right i will update